charts/consul/values.yaml

```diff
@@ -66,7 +66,7 @@ global:
   # image: "hashicorp/consul-enterprise:1.10.0-ent"
   # ```
   # @default: hashicorp/consul:<latest version>
-  image: docker.mirror.hashicorp.services/hashicorppreview/consul:1.18-dev
```

Comment:
~ Feels like we could start freezing these after CE is out of support since the tag should be equivalent

Reply:
makes sense, like not use -dev and just freeze at the CE tag?